models/users.js

```diff
@@ -215,6 +215,39 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Add default status
+ * @returns {Promise<{ count: number , updatedUsers: string[]}>}
+ */
+const addDefaultStatus = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const users = [];
+    const updateStatusPromises = [];
+    const updatedUsers = [];
+
+    usersSnapshot.forEach((doc) => users.push({ id: doc.id, ...doc.data() }));
+
+    for (const user of users) {
+      if (user.status === undefined) {
+        updateStatusPromises.push(
+          userModel.doc(user.id).update({
+            status: "idle",
```

Comment:
We can use the user status enum here from the constants file

Reply:
Added constant enum for user status. As the existing one was an array of user statuses, so doing `userStatusEnum[1]` for idle doesn't make sense

Reply:
> As the existing one was an array of user statuses, so doing userStatusEnum[1] for idle doesn't make sense

Then we should change the userStatusEnum to be an object like [profileStatus](https://github.com/Real-Dev-Squad/website-backend/blob/4f9c29dcce231be09e12b886587e2be74c485ca0/constants/users.js#L2)

Reply:
Added userStatus enum in the current implementation. Using that enum in existing implementation can have a separate issue.

Reply:
Adding the userStatus enum should also be done through a separate issue

Reply:
Created the issue https://github.com/Real-Dev-Squad/website-backend/issues/767

Reply:
done